src/hooks/useTable.js

```diff
@@ -206,7 +206,7 @@ export const useTable = (props, ...plugins) => {
         console.timeEnd('getAccessedRows')
       return [accessedData, rowPaths, flatRows]
     },
-    [debug, data, subRowsKey]
+    [debug, data, columns, subRowsKey]
```

Comment:
Is the columns variable used in this block?

Reply:
Nope, `instanceRef.current.columns` is.  

In this case it was being used as a flag to re-run the memo body when columns changes, but is mapping from the `instance.columns`.  

I could fix that.

Reply:
Let's do that. I would rather have the exhaustive deps lint rule do it and have our code reference the actual `columns` variable.